docs/nodes/mamaki-testnet.md

```diff
@@ -122,7 +127,8 @@ wget -O - https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
 
 ## Delegate to a validator
 
-To delegate tokens to the the `celestiavaloper` validator, as an example you can run:
+To delegate tokens to the the `celestiavaloper` validator, as an
```

Comment:
the the

Reply:
Fixed by [1302d84](https://github.com/celestiaorg/docs/pull/214/commits/1302d84a39b4a5b5c7f2c6e4ab3c2fdd50f977d4)